### ISSUE TITLE
Slimbook Titan -Lightbar not working  "no lighbar"

### ISSUE THREAD

**Issue by jcarolinares** [slimbook/slimbookrgbkeyboard#11]
Hello, I have one of the first Slimbook Titans made and besides some hardware problems I suspect my "custom ubuntu" for Titan never was properly updated (we had serious problems with the bootup at the time)

One of the things that never worked in Linux (windows no problem, so the lighbar is working) was the lighbar control with this program

![image](https://github.com/user-attachments/assets/68598235-6703-4400-8a99-df4305b8d4c8)

I've been launching the program in the terminal and it says:

```shell
jcarolinares@jcarolinares-TITAN:~$ slimbookrgbkeyboard
Slimbook TITAN detected
2024-08-10 11:55:17,682 - <module>:116 - INFO - Config check executed as jcarolinares
2024-08-10 11:55:17,682 - main:42 - INFO - Checking configuration
2024-08-10 11:55:17,682 - main:50 - INFO - Configuration folder (/home/jcarolinares/.config/slimbookrgbkeyboard) found!
2024-08-10 11:55:17,682 - main:53 - DEBUG - uid=1000 file_uid=1000
2024-08-10 11:55:17,682 - check_config_file:71 - INFO - Checking Slimbookrgbkeyboard's Configuration
2024-08-10 11:55:17,682 - check_config_file:81 - INFO - Checking section: CONFIGURATION ...
2024-08-10 11:55:17,682 - check_config_file:101 - INFO - Incidences not found.

Loading Keyboard options ...

Switch loaded: True
Scale loaded: 50.0
animations =['gaming_mode', 'gaming_mode_dead_light', 'gaming_mode_factorio', 'template.txt']
Config_effect: wave
Combo_effect: wave
Index: 1

Loading Lightbar options ...

Switch  loaded: True
Switch  loaded: False
 no lightbar
``` 

Any idea about what is happening here? It would be great to solve it

**Comment by Gabo-Tech** [comment]
I have the same issue, with lightbar and keyboard.

**Comment by Gabo-Tech** [comment]
also had serious problems with the bootup at the time.

**Comment by Lt-Henry** [comment]
> Hello, I have one of the first Slimbook Titans made and besides some hardware problems I suspect my "custom ubuntu" for Titan never was properly updated (we had serious problems with the bootup at the time)
> 
> One of the things that never worked in Linux (windows no problem, so the lighbar is working) was the lighbar control with this program
> 
> ![image](https://private-user-images.githubusercontent.com/7687624/356804201-68598235-6703-4400-8a99-df4305b8d4c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM1NTYyODAsIm5iZiI6MTcyMzU1NTk4MCwicGF0aCI6Ii83Njg3NjI0LzM1NjgwNDIwMS02ODU5ODIzNS02NzAzLTQ0MDAtOGE5OS1kZjQzMDViOGQ0YzgucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDgxMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA4MTNUMTMzMzAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZmI5NTQzM2RhOTM1MGMxM2NjYjU2Y2EzMDU0ZTBlZTQ1OTE0NDBjZjMyNjkzMDQ5NmI1YzI5MGY5MGQ0ZmM5ZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.TuGDAclqn07yTKPQ6WbjAKBCJpM_F7T2SzNVSF86gSY)
> 
> I've been launching the program in the terminal and it says:
> 
> ```shell
> jcarolinares@jcarolinares-TITAN:~$ slimbookrgbkeyboard
> Slimbook TITAN detected
> 2024-08-10 11:55:17,682 - <module>:116 - INFO - Config check executed as jcarolinares
> 2024-08-10 11:55:17,682 - main:42 - INFO - Checking configuration
> 2024-08-10 11:55:17,682 - main:50 - INFO - Configuration folder (/home/jcarolinares/.config/slimbookrgbkeyboard) found!
> 2024-08-10 11:55:17,682 - main:53 - DEBUG - uid=1000 file_uid=1000
> 2024-08-10 11:55:17,682 - check_config_file:71 - INFO - Checking Slimbookrgbkeyboard's Configuration
> 2024-08-10 11:55:17,682 - check_config_file:81 - INFO - Checking section: CONFIGURATION ...
> 2024-08-10 11:55:17,682 - check_config_file:101 - INFO - Incidences not found.
> 
> Loading Keyboard options ...
> 
> Switch loaded: True
> Scale loaded: 50.0
> animations =['gaming_mode', 'gaming_mode_dead_light', 'gaming_mode_factorio', 'template.txt']
> Config_effect: wave
> Combo_effect: wave
> Index: 1
> 
> Loading Lightbar options ...
> 
> Switch  loaded: True
> Switch  loaded: False
>  no lightbar
> ```
> 
> Any idea about what is happening here? It would be great to solve it

What version of Ubuntu are you using? is that jammy or noble?

**Comment by jcarolinares** [comment]
The info that I can find:

![image](https://github.com/user-attachments/assets/f9dd9e9a-9628-4bbb-a182-4d552ba99a4a)

So being version v22 I guess is Jammy

Is it possible to update my Titan to Ubuntu 24.04 without having driver related issues or loosing the Slimbook custom apps like the AMD controller and RGBkeyboard?

**Comment by Lt-Henry** [comment]
Yes, it is jammy. Are you using slimbook ppa? Can you provide slimbook packages version? ie: 

dpkg -l | grep slimbook

On the other hand, yes, we already moved apps and drivers to Ubuntu 24.04, however, slimbookbattery will no longer be available and slimbookface should be available soon, around september.

**Comment by jcarolinares** [comment]
Ok, the bash prompt:

```shell
jcarolinares@jcarolinares-TITAN:~$ dpkg -l | grep slimbook
ii  libslimbook1                                  1.14.1                                             amd64        Slimbook hardware identification library
ii  python3-slimbook                              1.14.0                                             amd64        libslimbook python binding
ii  slimbook-keyboard-dkms                        0.2.1                                              amd64        slimbook-keyboard driver in DKMS format.
ii  slimbook-qc71-dkms                            0.3.3                                              amd64        slimbook-qc71 driver in DKMS format.
ii  slimbookamdcontroller                         0.3.9beta                                          all          Aplicacion para la gestion de rendimiento de procesadores amd
ii  slimbookface                                  1.4.6                                              all          Slimbook Face
ii  slimbookgestures                              1.7beta                                            all          Slimbook Gestures
ii  slimbookmakeup                                1.1.20                                             all          Slimbook Makeup
ii  slimbookrgbkeyboard                           0.5.0                                              all          Slimbook RGB, gestiona el color y brillo de tu teclado.
ii  slimbooksensortouchpad                        0.3                                                all          set your touchpad
ii  slimbookwelcome                               1.9                                                all          Slimbook Welcome

``` 

Sounds great, I'm not in a hurry for the slimbookface one and I'm not using slimlboot battery so I will upgrade and test the face app around October!

**Comment by Lt-Henry** [comment]
By the way... did you open the laptop? (hardware upgrade, cleaning...) I am not on hardware side, but I think Lightbar is driven by a flex ribbon cable that may get disconnected accidentally.
